client/web/src/repo/blob/own/FileOwnershipPanel.tsx

```diff
@@ -18,18 +19,25 @@ import { FETCH_OWNERS } from './grapqlQueries'
 
 import styles from './FileOwnershipPanel.module.scss'
 
-export const FileOwnershipPanel: React.FunctionComponent<{
-    repoID: string
-    revision?: string
-    filePath: string
-}> = ({ repoID, revision, filePath }) => {
+export const FileOwnershipPanel: React.FunctionComponent<
+    {
+        repoID: string
+        revision?: string
+        filePath: string
+    } & TelemetryProps
+> = ({ repoID, revision, filePath, telemetryService }) => {
+    React.useEffect(() => {
```

Comment:
nit: Import `useEffect` to keep consistency with other hooks in this file

```suggestion
   useEffect(() => {
```

Reply:
Done! Thanks!